File: ueueo-framework/ueueo-uow/src/main/java/com/ueueo/uow/IAbpUnitOfWorkOptions.cs
using System;
using System.Data;

namespace Volo.Abp.Uow;

public interface IAbpUnitOfWorkOptions
{
    boolean IsTransactional;//  { get; }

    IsolationLevel? IsolationLevel;//  { get; }

    /**
     * Milliseconds
    */
    int? Timeout;//  { get; }
}

File: abp-frameworks/ueueo-uow/src/main/java/com/ueueo/uow/IAbpUnitOfWorkOptions.java
package com.ueueo.uow;

/**
 * @author Lee
 * @date 2022-05-19 20:35
 */
public interface IAbpUnitOfWorkOptions {

    boolean isTransactional();

    IsolationLevel getIsolationLevel();

    /**
     * Milliseconds
     *
     * @return
     */
    Integer getTimeout();
}
